### PR TITLE
Support for docker.types.Placement.MaxReplicas

### PR DESCRIPTION
**Pull request by WojciechowskiPiotr**
Hi,

Small PR adding support for Placement.MaxReplicas (new in API 1.40) in Docker Swarm Services. This was mentioned in https://github.com/docker/docker-py/issues/2528. There is also a feature request in ansible.docker community to support this feature (https://github.com/ansible-collections/community.docker/issues/7).

Signed-off-by: WojciechowskiPiotr <devel@it-playground.pl>

### CODE REVIEW

**Comment by muradm** [comment]
Wondering when this can be merged?

**Review by aiordache — APPROVED** [review]

File: docker/types/services.py [hunk 1]
```diff
@@ -659,10 +659,12 @@ class Placement(dict):
                 are provided in order from highest to lowest precedence and
                 are expressed as ``(strategy, descriptor)`` tuples. See
                 :py:class:`PlacementPreference` for details.
+            maxreplicas (int): Maximum number of replicas per node
             platforms (:py:class:`list` of tuple): A list of platforms
                 expressed as ``(arch, os)`` tuples
     """
-    def __init__(self, constraints=None, preferences=None, platforms=None):
+    def __init__(self, constraints=None, preferences=None, maxreplicas=None,
```
Comment (aiordache) [inline docker/types/services.py:666]: Can you add the new argument at the end please? This avoids breaking calls with positional args.

Comment (WojciechowskiPiotr) [reply docker/types/services.py]: Fixed as per your request, not sure why one of the tests failed this time

**Comment by ulyssessouza** [comment]
Hello @WojciechowskiPiotr .

Thank you for the contribution! Could you please add tests to your PR?

**Comment by WojciechowskiPiotr** [comment]
> Hello @WojciechowskiPiotr .
> 
> Thank you for the contribution! Could you please add tests to your PR?

Hi @ulyssessouza,

I've added integration and unit tests as per your request

**Pull request merged**